features/adults/conviction_motoring.feature

```diff
@@ -40,8 +40,19 @@ Feature: Adult Conviction
       | Fine                       | 01-01-2020 | Yes         | When were you given the fine?            | will be spent on 1 January 2025 |
       | Fine                       | 01-01-2020 | No          | When were you given the fine?            | will be spent on 1 January 2021 |
       | Fixed Penalty notice (FPN) | 01-01-2020 | Yes         | When was the endorsement given?          | will be spent on 1 January 2025 |
-      | Penalty points             | 01-01-2020 | Yes         | When were you given the penalty points?  | will be spent on 1 January 2025 |
-      | Penalty points             | 01-01-2020 | No          | When were you given the penalty points?  | will be spent on 1 January 2023 |
+
+  @happy_path
```

Comment:
Would be good to use the `@date_travel` and the `Given The current date is xxx` step here as well.

There are other cukes that will require it too but can be done as para of a separate PR as mentioned here: https://github.com/ministryofjustice/disclosure-checker/commit/184e8bb3c5f8b039a43b5e22744bdcd03d22d0f4

Reply:
Agreed. I have made a note and will address this in another PR.